### PR TITLE
Redesign UI update

### PR DESCRIPTION
**Pull request by marcuspeh**
Update logic to include `groups` list.
UI can now show both contacts view and groups view with a command or a click of a button.

### CODE REVIEW

File: src/main/java/seedu/address/ui/group/GroupListPanel.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ */
+public class GroupListPanel extends UiPart<Region> {
+    private static final String FXML = "GroupListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(GroupListPanel.class);
+
+    @FXML
+    private ListView<Group> groupListView;
+
+    /**
+     * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
```
Comment (Jingwencloud) [inline src/main/java/seedu/address/ui/group/GroupListPanel.java:25]: Did you mean GroupListPanel instead?

**Review by Jingwencloud — COMMENTED** [review]
Overall LGTM! Great work, just some very minor issues in the javadocs

File: src/main/java/seedu/address/logic/commands/ListGroupsCommand.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all persons in the address book to the user.
```
[the remark this answers is not in the document]
Comment (marcuspeh) [reply src/main/java/seedu/address/logic/commands/ListGroupsCommand.java:8]: Good catch! I didn't notice i missed that out

File: src/main/java/seedu/address/ui/group/GroupListPanel.java (hunk 1 above)
Comment (marcuspeh) [reply src/main/java/seedu/address/ui/group/GroupListPanel.java:25]: Yup, I have changed it

File: src/main/java/seedu/address/ui/NavigationButtonPanel.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.StackPane;
+
+
+/**
+ * The Main Window. Provides the basic application layout containing
```
Comment (ramapriyan912001) [inline src/main/java/seedu/address/ui/NavigationButtonPanel.java:9]: Perhaps you could make the docs here neater? Let a period be the end of a line.

File: src/main/java/seedu/address/ui/group/GroupCard.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.group.Group;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.ui.UiPart;
+
+
+/**
+ * An UI component that displays information of a {@code Person}.
```
Comment (ramapriyan912001) [inline src/main/java/seedu/address/ui/group/GroupCard.java:19]: do you mean group?

**Review by ramapriyan912001 — COMMENTED** [review]
Good job, a few minor issues with javadocs comments.

File: src/main/java/seedu/address/ui/NavigationButtonPanel.java (hunk 3 above)
Comment (marcuspeh) [reply src/main/java/seedu/address/ui/NavigationButtonPanel.java:9]: This is not the end of a sentence so there's no need for a period

File: src/main/java/seedu/address/ui/group/GroupCard.java (hunk 4 above)
Comment (marcuspeh) [reply src/main/java/seedu/address/ui/group/GroupCard.java]: I have updated it

**Pull request merged**